mkosi/__init__.py

```diff
@@ -32,6 +33,7 @@
 import urllib.parse
 import urllib.request
 import uuid
+from OpenSSL import crypto
```

Comment:
This means that we add the python openssl bindings as a first non-dev dependency. We can either do that, or move the import into the verity using functions and bail out if it's not there. 

The latter would preserve being (mostly) stdlib only, whereas the former could open the option of using other external libraries. In the former case some version constraints should be added to `setup.py`.

I'm not opining either way for or against the usage of external libraries, but I think this might warrant wider discussion.

Reply:
moving the import down where it is used sounds like a good solution to me

Reply:
I think it'd be nicer to something like 
```
try: from OpenSSL import crypto
except ImportError: crypto=None
```
and only error out after parsing args below if crypto would be used.